src/posts/2021-02-15-phaser-blog.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Phaser'
```

Comment:
Ik denk dat je de titel (voor SEO, en algemene leesbaarheid) wellicht is langer kan maken.

Bijvoorbeeld:
> Phaser, een gameframework voor HTML-5 games